sdk/iot/Azure.Iot.Hub.Service/src/Customized/Models/DigitalTwinInterfaces.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+
+namespace Azure.Iot.Hub.Service.Models
+{
+    /// <summary>
+    /// The DigitalTwinInterfaces.
+    /// </summary>
+    public partial class DigitalTwinInterfaces
+    {
```

Comment:
Do you need to create a new partial class? We should have PnpInterface defined already. I think you just need to add the property there. The naming was as per what we discussed yesterday.

Reply:
The property belongs to the DigitalTwinInterfaces class not the PnpInterfface class
PnpInterface is internal so we can't expose a property of that type with public accessibility

Reply:
am I missing something?

Reply:
I think we need to delete that class then. I might have made a mistake. There was no class called DigitalTwinInterfaces. There was only Interface.